src/templates/head.html

```
@@ -51,8 +51,8 @@
     </button>
     {% endif %}
     {# logo #}
-    <a href='dashboard.php' class='logo-header' title='{{ 'Go to dashboard'|trans }}' aria-label='{{ 'Go to dashboard'|trans }}'>
-      {{ App.Config.configArr.logo_header_svg|raw }}
+    <a href='dashboard.php' title='{{ 'Go to dashboard'|trans }}' aria-label='{{ 'Go to dashboard'|trans }}'>
+       <img class='logo-header' src='/api/v2/instance/branding/{{ constant('Elabftw\\Enums\\Branding::Header').value }}?format=binary' alt='eLabFTW'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)